bleak/exc.py

```diff
@@ -16,18 +16,34 @@ class BleakDotNetTaskError(BleakError):
 class BleakDBusError(BleakError):
     """Specialized exception type for D-Bus errors."""
 
-    def __init__(self, dbus_error: str):
+    def __init__(self, dbus_error: str, error_body: list = None):
```

Comment:
Does `error_body` still need to be optional?

Reply:
No, that is left over because I first thought this class is instantiated in multiple places.